cypress/component/integration/Clipboard.cy.tsx

```diff
@@ -0,0 +1,174 @@
+        });
+
+        cy.get('[data-testid="clipboard-field"]').click();
+        cy.get('@copyToClipboardPrompt').should('be.called');
+
+        cy.get('@copyToClipboardPrompt')
+          .should((prompt) => {
+            expect(prompt.args[0][1]).to.equal('raw content');
+          })
+          .then(() => {
+            cy.get('[data-testid="clipboard-input-message"]')
+              .should('exist')
+              .contains('Copied!');
+          });
+
+        cy.get('[data-testid="clipboard-input-message"]').should('not.exist');
+      });
+
+      it('should match snapshot', () => {
+        compareSnapshot(cy, 'clipboard-success-flow');
+      });
+    });
+
+    describe('clipboard raw content success flow', () => {
+      beforeEach(() => {
+        cy.mount(<Clipboard rawContent={'raw content'} success="Copied!" />);
+      });
+      it('should render', () => {
+        cy.get('[data-testid="clipboard-field"]').should('exist');
+        cy.get('[data-testid="clipboard-content"]').contains('raw content');
+      });
+
+      it('should fire the copy clipboard function', () => {
+        cy.get('[data-testid="clipboard-field"]').should('exist');
+        cy.get('[data-testid="clipboard-content"]').contains('raw content');
+        cy.on('window:confirm', () => true);
+
+        cy.window().then((win) => {
+          cy.stub(win, 'prompt')
+            .returns(win.prompt)
+            .as('copyToClipboardPrompt');
+        });
+
+        cy.get('[data-testid="clipboard-field"]').click();
+        cy.get('@copyToClipboardPrompt').should('be.called');
+
+        cy.get('@copyToClipboardPrompt')
+          .should((prompt) => {
+            expect(prompt.args[0][1]).to.equal('raw content');
+          })
+          .then(() => {
+            cy.get('[data-testid="clipboard-input-message"]')
+              .should('exist')
+              .contains('Copied!');
+          });
+
+        cy.get('[data-testid="clipboard-input-message"]').should('not.exist');
+      });
+
+      it('should match snapshot', () => {
+        compareSnapshot(cy, 'clipboard-raw-content-success-flow');
+      });
+    });
+
+    describe('clipboard error flow', () => {
+      beforeEach(() => {
+        cy.mount(<Clipboard rawContent={''} error="Oupps!" />);
+      });
+      it('should render', () => {
+        cy.get('[data-testid="clipboard-field"]').should('exist');
+        cy.get('[data-testid="clipboard-content"]')
+          .contains('foo')
+          .should('not.exist');
+      });
+
+      it('should not fire the copy clipboard function', () => {
+        cy.get('[data-testid="clipboard-field"]').should('exist');
+        cy.get('[data-testid="clipboard-content"]')
+          .contains('foo')
+          .should('not.exist');
+        cy.on('window:confirm', () => true);
+
+        cy.window().then((win) => {
+          cy.stub(win, 'prompt')
+            .returns(win.prompt)
+            .as('copyToClipboardPrompt');
+        });
```

Comment:
If we test that already before we don't need to repeat it here. Just the aim of the test. I mean, does not need to assert against what you already asserted in a test before. If it fails, it will fails in the previous test.

Reply:
here we are testing that it is not fired